src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1041 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Navigation;
+using Windows.UI.Xaml.Media;
+using System.Collections.ObjectModel;
+
+using TreeViewSelectionMode = Microsoft.UI.Xaml.Controls.TreeViewSelectionMode;
+using TreeViewNode = Microsoft.UI.Xaml.Controls.TreeViewNode;
+using TreeView = Microsoft.UI.Xaml.Controls.TreeView;
+using TreeViewItemInvokedEventArgs = Microsoft.UI.Xaml.Controls.TreeViewItemInvokedEventArgs;
+using TreeViewExpandingEventArgs = Microsoft.UI.Xaml.Controls.TreeViewExpandingEventArgs;
+using TreeViewDragItemsStartingEventArgs = Microsoft.UI.Xaml.Controls.TreeViewDragItemsStartingEventArgs;
+using TreeViewDragItemsCompletedEventArgs = Microsoft.UI.Xaml.Controls.TreeViewDragItemsCompletedEventArgs;
+using TreeViewList = Microsoft.UI.Xaml.Controls.TreeViewList;
+using TreeViewItem = Microsoft.UI.Xaml.Controls.TreeViewItem;
+//using MaterialHelperTestApi = Microsoft.UI.Private.Media.MaterialHelperTestApi;
+using System.Threading.Tasks;
+
+// Uno specific
+using Uno.UI.Samples.Controls;
+using MUXControlsTestApp.Utilities;
+
+namespace UITests.Shared.Microsoft_UI_Xaml_Controls.TreeViewTests
+{
+	[SampleControlInfo("TreeView", "MUX_Test_TreeViewPage")]    
+    public sealed partial class TreeViewPage : MUXTestPage
+    {
+		bool _disableClickToExpand;
+		public TreeViewItem flyoutTVI;
+		TreeViewNode _visualRoot;
+		TreeViewNode _virtualizedNode;
+		public ObservableCollection<TreeViewItemSource> TestTreeViewItemsSource { get; set; }
+		public ObservableCollection<TreeViewItemSource> TestTreeView2ItemsSource { get; set; }
+
+		public TreeViewPage()
+		{
+			this.DataContext = this;
+			this.InitializeComponent();
+
+			//TODO:
+			//MaterialHelperTestApi.IgnoreAreEffectsFast = true;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)